mixins/eirinix/templates/dns-aliases/deployment.yaml

```diff
@@ -0,0 +1,42 @@
+{{- $_ := include "_config.load" $ }}
+{{- if .Values.features.eirini.enabled }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: eirini-dns-aliases
+  labels:
+    {{- list . "eirini-dns-aliases" | include "component.labels" | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- list . "eirini-dns-aliases" | include "component.selector" | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- list . "eirini-dns-aliases" | include "component.labels" | nindent 8 }}
+    spec:
+      serviceAccountName: eirinix
+      containers:
+        - name: eirini-dns-aliases
+          {{- with index .Values.eirinix "dns-aliases" }}
+          image: {{ printf "%s:%s" .image .image_tag | quote }}
+          imagePullPolicy: {{ .pullPolicy }}
+          {{- end }}
+          env:
+          - name: EIRINI_NAMESPACE
+            value: {{ .Values.eirini.opi.namespace }}
+          - name: WEBHOOK_NAMESPACE
+            value: {{ .Release.Namespace }}
+          - name: PORT
+            value: "8443"
+          - name: SERVICE_NAME
+            value: eirini-dns-aliases
+          - name: OPERATOR_FINGERPRINT
```

Comment:
This is something that should be hardcoded in the binary and not something we should edit later on - it causes mutatingwebhooks and secrets to change names, which probably would break during upgrades